src/pages/tutorials/build-an-nft-marketplace/index.md

```diff
@@ -201,40 +196,89 @@ Explanations:
   - `NFT.Metadata.t` : tzip-16 compliance
   - `NFT.TokenMetadata.t` : tzip-12 compliance
   - `NFT.Operators.t` : permissions part of FA2 standard
-  - `NFT.Storage.token_id>` : cache for keys of token_id bigmap
+  - `set<NFT.token_id>` : cache for keys of token_id bigmap
 - `storage` has more fields to support a set of `administrators`
-- `parameter` definition is an extension of the imported library entrypoints
-  - `NFT.transfer` : to transfer NFTs
-  - `NFT.balance_of` : to check token balance for a specific user (on this template it will return always 1)
-  - `NFT.update_operators` : to allow other users to manage our NFT
-- `parameter` has more entrypoints to allow to create NFTs `Mint`
-- `parameter` has an entrypoint `AddAdministrator` to add new administrators. Administrators will be allowed to mint NFTs
-
-Compile the contract
 
-```bash
-TAQ_LIGO_IMAGE=ligolang/ligo:0.60.0 taq compile nft.jsligo
-```
+The contract compiles, now let's write `transfer,balance_of,update_operators` entrypoints. We will do a passthrough call to the underlying library.
```

Comment:
Make it clear what the user is doing: pasting in code. Seems strange to put a comma-separated list in backticks.